aslam_backend/src/LinearSystemSolver.cpp

```diff
@@ -65,19 +68,19 @@ namespace aslam {
           indices[i + 1] = indices[i] + nJPerThread;
         // deal with the remainder.
         indices.back() = _errorTerms.size();
-        // Build a thread pool and evaluate the jacobians.
-        boost::thread_group threads;
-        std::vector<SafeJob> jobs(nThreads);
```

Comment:
Awesome! It might even be that we can remove the `SafeJob` completely if it isn't used anywhere else in this file.